.github/workflows/icon-review.yml

```diff
@@ -3,107 +3,122 @@ name: 🏞️ Icon Review
 on:
   pull_request_target:
     paths:
+      # Only trigger the workflow if SVG files in the icons directory are changed
       - "icons/*.svg"
 
+# Permissions for the workflow
 permissions:
   contents: read
+  pull-requests: write
 
 jobs:
   icon-review:
     name: Icon Review
     runs-on: ubuntu-latest
-    env:
-      TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
-
-    permissions:
-      pull-requests: write
 
     steps:
-      - name: 📥 Checkout Fork
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-          fetch-depth: 0
-          path: fork
-          persist-credentials: false
-
-      - name: 📥 Checkout Original
+      - name: 📥 Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-          path: main
           persist-credentials: false
```

Comment:
Does this still work on forks?

Otherwise, this diff is big, but a brief skim said there wasn't anything glaring.

Reply:
Had the same thought while going through the changes. There was a big struggle with the forks. Would be great if it would work as suggested in this PR, but it would be better to test it before.

Reply:
It only responds to pull requests in this repository, so I think everything is correct

Reply:
I just tested it and it didn't work. I've updated the color of the Angular icon to purple:

https://pixp.pages.dev/pkief/vscode-material-icon-theme-labs/pull/6

But the generated preview image looks like that:

https://github.com/PKief/vscode-material-icon-theme-labs/pull/6#issuecomment-2257885308

Reply:
> I just tested it and it didn't work. I've updated the color of the Angular icon to purple:
> 
> https://pixp.pages.dev/pkief/vscode-material-icon-theme-labs/pull/6
> 
> But the generated preview image looks like that:
> 
> https://github.com/PKief/vscode-material-icon-theme-labs/pull/6#issuecomment-2257885308

Hmm... It's strange, I will analyze the reason

Reply:
Apparently it updates the comment incorrectly

Reply:
Apparently it updates the comment incorrectly